src/components/Navbar.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import BoardIcon from "../assets/icons/BoardIcon";
+import DashboardIcon from "../assets/icons/DashboardIcon";
+import HomeIcon from "../assets/icons/HomeIcon";
+import SettingsIcon from "../assets/icons/SettingsIcon";
+import LogoutIcon from "../assets/icons/LogoutIcon";
+import { useLocation, Link } from "react-router-dom";
+import DefaultAvatar from "../assets/images/default-avatar.png";
+
+function Navbar(props) {
+  const [hovering, setHovering] = useState(false);
+  const location = useLocation();
+  return (
+    <div className="navbar">
+      <div className="nav-icongroup-container">
+        <Link to={"/home"} className="button-link">
+          <div className="nav-icon-container">
+            {location.pathname === "/home" && <div className="selected-box" />}
+            <HomeIcon />
+          </div>
+        </Link>
+        <Link to={"/dashboard"} className="button-link">
+          <div className="nav-icon-container">
+            {location.pathname === "/dashboard" && (
+              <div className="selected-box" />
+            )}
+            <DashboardIcon />
+          </div>
+        </Link>
+        <Link to={"/board"} className="button-link">
+          <div className="nav-icon-container">
+            {location.pathname === "/board" && <div className="selected-box" />}
+            <BoardIcon />
+          </div>
+        </Link>
+      </div>
+
+      <div
+        className="account-container"
```

Comment:
I dont think we should only be displaying icons on hovering. It can make it very confusing for people to see where they should go to access certain buttons. It will also be an issue for mobile users, as the hover functionality doesn't really apply

Reply:
I disagree. It's pretty standard across all platforms to show the logout and configure buttons/icons when you click on the user avatar.
And for mobile, you can just tap it like every other app